src/Platforms/CUDA/CUDALinearAlgebraHandles.h

```diff
@@ -15,28 +15,27 @@
 #include "ResourceCollection.h"
 #include "CUDA/CUDAruntime.hpp"
 #include "CUDA/cuBLAS.hpp"
+#include "CUDA/QueueCUDA.hpp"
 
 namespace qmcplusplus
 {
 struct CUDALinearAlgebraHandles : public Resource
 {
   // CUDA specific variables
-  cudaStream_t hstream;
+  compute::Queue<PlatformKind::CUDA> queue;
```

Comment:
feels like now might be the time to change the name of this class. the minute the sycl queue shows up the error messages are all going to be sometimes wrong as well.

Reply:
Were you referring to CUDALinearAlgebraHandles? Yes it has to be changed. It is not clear how exactly. I probably will demolish this class. So far it is still CUDA only and should not be renamed in this PR.

Reply:
Yes I was referring to this.